api/.env.example

```diff
@@ -84,11 +84,15 @@ FRESHDESK_API_TICKET_TAG=
 FRESHDESK_API_PRODUCT_ID=
 FRESHDESK_API_EMAIL_CONFIG_ID=
 
-
 # GC Notify API
 GCNOTIFY_API_KEY=
 GCNOTIFY_TEMPLATE_TEST_EMAIL="c386053b-e65c-477e-8f09-da19c2e9dea0"
 GCNOTIFY_TEMPLATE_TEST_SMS="3525f0e3-845a-4001-b456-7cb02c13c69f"
 GCNOTIFY_TEMPLATE_TEST_BULK_EMAIL="90fa07f5-fbe8-402e-9547-f3259159a2da"
 GCNOTIFY_TEMPLATE_TEST_BULK_SMS="d662aac4-08bb-4917-860f-cc9999f4648d"
 
+# Feature flags
+FEATURE_ONGOING_RECRUITMENTS=true
+FEATURE_APPLICANT_DASHBOARD=true
+FEATURE_APPLICATION_REVAMP=true
+FEATURE_PSAC_STRIKE=true
```

Comment:
Just thinking here, nothing to action. But, I really wish we had  central location for these in development 😓

Reply:
Funny you should mention that...
https://github.com/GCTC-NTGC/gc-digital-talent/discussions/6375